package.json

```diff
@@ -7,7 +7,7 @@
     "start-original": "ng serve",
     "start": "ng serve --verbose --proxy-config proxy.conf.json",
     "build": "ng build --prod",
-    "test": "ng test",
+    "test": "ng test -sm=false",
```

Comment:
Try running tests without this flag and see if the error still persists. If no, remove the flag.